containers/ssh/Dockerfile

```diff
@@ -1,3 +1,4 @@
-FROM sickp/alpine-sshd:latest
```

Comment:
This will remove sshd which is the point of this container. 

Its also for local testing/demo and not recommended to actually use

Reply:
oh, I see. 
sorry for carelessness to not import sshd. I was using the docker image to login to host of docker container.
I understood that it's for testing/demo.